cla_public/static-templates/errors/50x.html

```diff
@@ -121,51 +121,51 @@
         <div class="govuk-phase-banner">
           <p class="govuk-phase-banner__content">
             <span class="govuk-phase-banner__text" id="aria-feedback">
-              <a href="{{ url_for('base.reasons_for_contacting')}}">Report a problem</a> if you need help or would like to give feedback to improve this service
+              <a href="{{ url_for('base.reasons_for_contacting')}}">{% trans %}Report a problem</a> if you need help or would like to give feedback to improve this service{% endtrans %}
```

Comment:
Are we sure that, in Welsh, the linked part of the text will always be at the front?  If not - we should include the tag in the translation file.  
`{% trans %}<a href="{{ url_for('base.reasons_for_contacting')}}">Report a problem</a> if you need help or would like to give feedback to improve this service{% endtrans %}
That way, the Welsh could be xxxxxxxxx <a>xxxxxxx</a> xxxxxxx if need be.
            `